extensions/cornerstone/src/index.tsx

```diff
@@ -34,6 +35,13 @@ const OHIFCornerstoneViewport = props => {
   );
 };
 
+const registerCacheService = {
```

Comment:
The cache is a global singleton, so register it this way.

Reply:
Can you move this to its own file similar to `ToolGroupService` and the rest?

Reply:
The issue is that it is accessed both statically and as a real service.  How about I move the export object into the cache export as an additional export?